### PR TITLE
enable compression by default

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: webapp/src/main/resources/application.yml [hunk 1]
```diff
@@ -24,12 +24,15 @@ saml:
...
-# tomcat settings for reasonable performance: adjust for tiny/huge configurations
 server:
+  # tomcat settings for reasonable performance: adjust for tiny/huge configurations
   tomcat:
     max-connections: 0
     max-threads: 4
     accept-count: 10
+  compression:
+    enabled: true
+    mime-types: application/json,application/xml,text/html,text/xml,text/plain,text/css,text/javascript,application/javascript
```
Comment (malaffoon) [inline webapp/src/main/resources/application.yml]: just copied this list from what Will did: good?

Comment (bgsavage) [reply webapp/src/main/resources/application.yml]: Just curious if there's a mime-type we don't want to support compression on, or just allow it to do all?

Also do we even serve xml?

Comment (wtritch) [reply webapp/src/main/resources/application.yml]: lgtm.  I don't think we do today, but having it in there doesn't hurt anything.

Comment (bgsavage) [reply webapp/src/main/resources/application.yml]: Why not just wildcard it then?

Comment (bgsavage) [reply webapp/src/main/resources/application.yml]: Actually ignore that, I don't think you can...

Comment (esotrope) [reply webapp/src/main/resources/application.yml]: what about csv?

Comment (malaffoon) [reply webapp/src/main/resources/application.yml]: Noted. I patched the code.

**Pull request merged**